### PR TITLE
feat(docs): add examples and tutorials page

### LINKED ISSUE #754: where should we put more complex examples
I built an example of a rails app with all of the draft bits [here](https://github.com/michelleN/todo-app)

It's a bit larger than our examples though, so I'm thinking we may not want to actually merge this into the draft repo. Instead, should we have a doc that points to these kinds of examples in the docs?

### PR DESCRIPTION
**Pull request by michelleN**
resolves #754

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ This guide covers how you can quickly get started using Draft
 The following prerequisites are required for successful use of Draft.
 
 1. A Kubernetes cluster and the `kubectl` CLI tool
-2. Installation of Helm on your Kubernetes cluster
+2. Installation of [Helm](https://github.com/Kubernetes/helm) on your Kubernetes cluster
```
Comment (bacongobbler) [inline docs/quickstart.md]: should kubernetes be lowercase?

Comment (michelleN) [reply docs/quickstart.md]: I don't actually know. Let me look around!

Comment (michelleN) [reply docs/quickstart.md]: It looks like in other places in our docs and comments, it is capitalized.

Comment (michelleN) [reply docs/quickstart.md]: I'll leave as is to preserve consistency but open to changing in the future if necessary!

**Pull request merged**